web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -199,36 +199,38 @@ import {get_string_diff} from "../../src/util";
   }
 
   , show: function () {
+    if (!this.options.doNotMove) {
       var pos;
 
-      if (this.fixed) {
-        // Relative to screen instead of to page
-        pos = this.$element[0].getBoundingClientRect();
-      } else {
-        pos = this.$element.offset();
-      }
+        if (this.fixed) {
+          // Relative to screen instead of to page
+          pos = this.$element[0].getBoundingClientRect();
+        } else {
+          pos = this.$element.offset();
+        }
 
-      pos = $.extend({}, pos, {
-        height: this.$element[0].offsetHeight
-      })
+        pos = $.extend({}, pos, {
+          height: this.$element[0].offsetHeight
+        })
 
-      // Zulip patch: Workaround for iOS safari problems
-      pos.top = this.$element.offset().top;
+        // Zulip patch: Workaround for iOS safari problems
+        pos.top = this.$element.offset().top;
 
-      var top_pos = pos.top + pos.height
-      if (this.dropup) {
-        top_pos = pos.top - this.$container.outerHeight()
-      }
+        var top_pos = pos.top + pos.height
+        if (this.dropup) {
+          top_pos = pos.top - this.$container.outerHeight()
+        }
 
-      // Zulip patch: Avoid typeahead going off top of screen.
-      if (top_pos < 0) {
-          top_pos = 0;
-      }
+        // Zulip patch: Avoid typeahead going off top of screen.
+        if (top_pos < 0) {
+            top_pos = 0;
+        }
 
-      this.$container.css({
-        top: top_pos
-       , left: pos.left
-      })
+        this.$container.css({
+          top: top_pos
+         , left: pos.left
+        })
+      }
```

Comment:
Changes to the forked typeahead library should always be done in independent commits that includes updating the documentation at the top of the file and has a commit message explaining why it's necessary. I'm not sure I understand what you're doing here, but in any case, @aero31aero @N-Shar-ma have been working on the typeahead code much more than I have, and probably should review this part.

Reply:
Thanks for the review @N-Shar-ma! Tim, I added the documentation and a note to the commit message. I'm not sure if it makes sense to separate the change into its own commit -- see my comment below.